plugins/ocm/README.md

```diff
@@ -179,146 +189,141 @@ This plugin is made up of 2 packages:
+     +             </ClusterContextProvider>
+     +           </EntitySwitch.Case>
+         ...
+     + }
+       ...
+       export const entityPage = (
+         <EntitySwitch>
+           ...
+     +     <EntitySwitch.Case if={isKind('resource')} children={resourcePage} />
+           ...
+         </EntitySwitch>
+       );
+
+
+     ```
+
+     In the previous code, you can place the context provider into your `Resource` entity renderer, which is usually available in `packages/app/src/components/catalog/EntityPage.tsx` or in an imported component.
+
+   - `ClusterStatusCard`: This is an entity component displaying availability status of a cluster as an overview card:
+
+     ```diff
+       <ClusterContextProvider>
+         ...
+     +   <ClusterStatusCard />
+         ...
+       </ClusterContextProvider>
+     ```
+
+   - `ClusterInfoCard`: This is an entity component displaying details of a cluster in a table:
+
+     ```diff
+       <ClusterContextProvider>
+         ...
+     +   <ClusterInfoCard />
+         ...
+       </ClusterContextProvider>
+     ```
+
+   - `ClusterAvailableResourceCard`: This is an entity component displaying the available resources on a cluster. For example, see [`.status.capacity`](https://open-cluster-management.io/concepts/managedcluster/#cluster-heartbeats-and-status) of the `ManagedCluster` resource.
+
+     ```diff
+       <ClusterContextProvider>
+         ...
+     +   <ClusterAvailableResourceCard />
+         ...
+       </ClusterContextProvider>
+     ```
+
+   - `ClusterAllocatableResourceCard`: This is an entity component displaying allocatable resources on a cluster. For example, see [`.status.allocatable`](https://open-cluster-management.io/concepts/managedcluster/#cluster-heartbeats-and-status) of the `ManagedCluster` resource.
+
+     ```diff
+       <ClusterContextProvider>
+         ...
+     +   <ClusterAllocatableResourceCard />
+         ...
+       </ClusterContextProvider>
+     ```
 
-```diff
-  <ClusterContextProvider>
-    ...
-+   <ClusterAvailableResourceCard />
-    ...
-  </ClusterContextProvider>
-```
+## Development
 
-#### ClusterAllocatableResourceCard
+If you have installed the OCM plugin to the example application in the repository, then run `yarn start` command to access the plugin in the root directory and then navigate to [/ocm](http://localhost:3000/ocm).
 
-Entity component showing allocatable resources on the cluster. References [`.status.allocatable`](https://open-cluster-management.io/concepts/managedcluster/#cluster-heartbeats-and-status) of the `ManagedCluster` resource.
+To start a development setup in isolation with a faster setup and hot reloads, follow the following steps:
 
-```diff
-  <ClusterContextProvider>
-    ...
-+   <ClusterAllocatableResourceCard />
-    ...
-  </ClusterContextProvider>
-```
+1. Run the `ocm-backend` plugin in the `plugins/ocm-backend` directory by executing the following command:
```

Comment:
use consistent language - run vs execute vs enter vs use when referring to commands. I would go with "run the following command" throughout